Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelementryDiagnosticControlss.java

```diff
@@ -0,0 +1,29 @@
+package tddmicroexercises.telemetrysystem;
+
+public class TelementryDiagnosticControlss implements DiagnosticControls {
```

Comment:
Their is another class with similar name TelemetryDiagnosticControls
Please remove unused one

Actually both the classes are redundant
You can directly use TelemetryClient which implements DiagnosticControls